src/gwt/src/org/rstudio/studio/client/server/remote/RemoteServer.java

```diff
@@ -184,6 +184,10 @@
 import com.google.inject.Provider;
 import com.google.inject.Singleton;
 
+import elemental.json.JsonNumber;
```

Comment:
I think these may have been imported by mistake (we don't use the elemental.json wrappers in our marshaling)